llama_stack/providers/remote/inference/ollama/ollama.py

```diff
@@ -107,7 +114,40 @@ async def initialize(self) -> None:
         logger.debug(f"checking connectivity to Ollama at `{self.url}`...")
         health_response = await self.health()
         if health_response["status"] == HealthStatus.ERROR:
-            raise RuntimeError("Ollama Server is not running, start it using `ollama serve` in a separate terminal")
+            logger.warning(
+                "Ollama Server is not running, make sure to start it using `ollama serve` in a separate terminal"
+            )
+
+        if self.refresh_models:
+            # periodically refresh the model list
+            asyncio.create_task(self._refresh_models())
+
+    async def _refresh_models(self) -> None:
+        if not self.model_store:
+            raise ValueError("Model store not set")
+        provider_id = self.__provider_id__
+        while True:
+            try:
+                provider_models = await self.client.list()
+            except Exception as e:
+                logger.warning(f"Failed to list models: {str(e)}")
+                await asyncio.sleep(REFRESH_MODELS_INTERVAL)
+                continue
+
+            models = [
+                Model(
+                    identifier=m.model,
+                    provider_resource_id=m.model,
+                    provider_id=provider_id,
+                    metadata={},
+                    model_type=ModelType.llm,
+                )
+                for m in provider_models
+            ]
+
+            await self.model_store.update_registered_models(provider_id, models)
+            logger.info("[ollama] refreshed model list")
```

Comment:
should this be info or debug? Info might clutter the logs a bit especially If multiple providers might be doing this